### ISSUE TITLE
[bug] No respect to markdown anymore for dairy file

### ISSUE THREAD

**Issue by itsNikolay** [vimwiki/vimwiki#463]
After upgrade Vimwiki **doesn't respect markdown extension** config for **current day daily file**
It opens `diary/2018-04-02.wiki` instead of `diary/2018-03-30.md`
But I have in my `.vimrc` file:
```vim
" Vimwiki
let g:vimwiki_list = [{'path': '~/vimwiki/', 'syntax': 'markdown', 'ext': '.md'}]
```

It worked well on few month ago version

**Comment by EinfachToll** [comment]
I think this is solved in the meantime by #442. Update Vimwiki and check again.

**Comment by itsNikolay** [comment]
@EinfachToll thanks, it works now correct!
